compute/wps/webapp/src/app/configure/process-detail.component.ts

```diff
@@ -163,16 +163,36 @@ export class ProcessDetailComponent implements OnInit {
     });
   }
 
+  getSelectedFiles() {
+    if (this.files == null) { throw 'No files have been selected'; }
```

Comment:
Does a check for null also catch if the variable is undefined? I cant remember ...

Reply:
This will be completely changed in the workflow branch so I wouldn't worry about it.